diff --git a/node_modules/expo-firebase-core/android/build.gradle b/node_modules/expo-firebase-core/android/build.gradle
index 9bda09b..cf1b1ab 100644
--- a/node_modules/expo-firebase-core/android/build.gradle
+++ b/node_modules/expo-firebase-core/android/build.gradle
@@ -35,11 +35,7 @@ buildscript {
   }
 }
 
-// Creating sources with comments
-task androidSourcesJar(type: Jar) {
-  classifier = 'sources'
-  from android.sourceSets.main.java.srcDirs
-}
+
 
 afterEvaluate {
   publishing {
